### PR TITLE
create pt-br language

### PR DESCRIPTION
**Pull request by Swellington-Soares**
## Checklist

- [x] I have personally loaded this code into an updated Qbox project and checked all of its functionality.
- [x] My pull request fits the contribution guidelines & code conventions.

### CODE REVIEW

File: locales/pt-br.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
+    "error": {
+        "no_space": "Outro veículo já ocupou o espaço.",
+        "no_access": "Você não tem acesso a esta garagem",
+        "no_vehicles": "Você atualmente não tem veículos neste local...",
+        "not_impound": "Seu veículo não está no pátio",
```
Comment (rust1co) [inline locales/pt-br.json:6]:
```suggestion
        "not_impound": "Você não possui nenhum veículo apreendido",
```

File: locales/pt-br.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        "not_owned": "Este veículo não pode ser guardado",
```
Comment (rust1co) [inline locales/pt-br.json:7]:
```suggestion
        "not_owned": "Você não é o dono deste veiculo",
```

File: locales/pt-br.json [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+        "not_correct_type": "Você não pode guardar este tipo de veículo aqui",
+        "not_enough": "Dinheiro insuficiente",
+        "vehicle_occupied": "Você não pode guardar este veículo pois ele não está vazio",
+        "in_vehicle": "Você já está em um veículo...",
+        "spawn_in_progress": "Por favor, aguarde enquanto seu veículo é entregue..."
+    },
+    "success": {
+        "vehicle_parked": "Veículo guardado"
```
Comment (rust1co) [inline locales/pt-br.json:15]:
```suggestion
        "vehicle_parked": "Você estacionou seu veículo"
```

**Review by rust1co — COMMENTED** [review]
Making the words make more sense in Brazilian Portuguese

**Review by D4isDAVID — APPROVED** [review]

**Pull request merged**